File: datasets_v2/media/awkwardratings/modify/fragment_intro.swift
import SwiftUI
struct fragment_intro: View {

var body: some View {
ZStack
{
VStack
{
Text("Welcome to...")
.padding(.bottom,16)
.font(.system(size: 14))

Text("Awkward Ratings")
.padding(.bottom,16)
.font(.system(size: 34))

Text("the app where you find out which movies\nwould be awkward to watch with your parents...")
.padding(.bottom,16)
.font(.system(size: 14))

Text("...or the app where you find out which movies\nwould be awkward to watch with your kids.")
.padding(.bottom,16)
.font(.system(size: 14))

Text("Whichever floats your boat.")
.padding(.bottom,16)
.font(.system(size: 14))

Text("Awkward ratings for each movie\nare done by you, our users...")
.padding(.bottom,16)
.font(.system(size: 14))

Text("...and a warehouse full of movie elves.")
.padding(.bottom,16)
.font(.system(size: 14))

Text("So go help those elves out...")
.padding(.bottom,16)
.font(.system(size: 14))

Text("")
.padding(.bottom,16)
Text("...or not.")
.padding(.bottom,16)
.font(.system(size: 14))
Text("- Nasah Apps")
.padding(.bottom,16)
.font(.system(size: 18))

Button(action: {})
{
Text("Let's get started")
.foregroundColor(.black)
}
.background(Color.white)
}
.frame(minWidth:0,maxWidth:.infinity)
}
.frame(minWidth:0,maxWidth:.infinity,minHeight:0,maxHeight:.infinity)
.padding(.top,16)
.padding(.bottom,16)
.padding(.leading,16)
.padding(.trailing,16)}}

struct fragment_intro_Previews: PreviewProvider {
    static var previews: some View {
        fragment_intro()
    }
}

File: experiments/media/awkwardratings/compose/fragment_intro.kt
import androidx.compose.ui.tooling.preview.Preview
import androidx.compose.foundation.background
import androidx.compose.foundation.layout.*
import androidx.compose.foundation.lazy.LazyColumn
import androidx.compose.foundation.lazy.items
import androidx.compose.material3.*
import androidx.compose.runtime.*
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.paint
import androidx.compose.ui.graphics.Color
import androidx.compose.foundation.Image
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.layout.ContentScale
import androidx.compose.ui.text.style.TextAlign
import androidx.compose.ui.unit.sp
import androidx.compose.ui.unit.dp

@Composable
fun fragment_intro() {
  Column(modifier = Modifier.fillMaxWidth().fillMaxHeight().padding(top = 16.0.dp).padding(bottom = 16.0.dp).verticalScroll(rememberScrollState())) {
    Column(modifier = Modifier.fillMaxWidth().wrapContentHeight(), horizontalAlignment = Alignment.End) {
      Text(text = "Welcome to...", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(bottom = 16.0.dp), fontSize = 14.sp)
      Box(modifier = Modifier.fillMaxWidth(), contentAlignment = Alignment.TopCenter) {
      Text(text = "Awkward Ratings", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(bottom = 16.0.dp), fontSize = 34.sp)
      }
      Text(text = "the app where you find out which movies\\nwould be awkward to watch with your parents...", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(bottom = 16.0.dp), fontSize = 14.sp)
      Text(text = "...or the app where you find out which movies\\nwould be awkward to watch with your kids.", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(bottom = 16.0.dp), fontSize = 14.sp, textAlign = TextAlign.End)
      Text(text = "Whichever floats your boat.", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(bottom = 16.0.dp), fontSize = 14.sp)
      Text(text = "Awkward ratings for each movie\\nare done by you, our users...", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(bottom = 16.0.dp), fontSize = 14.sp, textAlign = TextAlign.End)
      Text(text = "...and a warehouse full of movie elves.", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(bottom = 16.0.dp), fontSize = 14.sp)
      Text(text = "So go help those elves out...", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(bottom = 16.0.dp), fontSize = 14.sp, textAlign = TextAlign.End)
      Text(text = "", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(bottom = 16.0.dp))
      Text(text = "...or not.", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(bottom = 16.0.dp), fontSize = 14.sp)
      Text(text = "- Nasah Apps", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(bottom = 16.0.dp), fontSize = 18.sp, textAlign = TextAlign.End)
      Button(onClick = {}, enabled = true, modifier = Modifier.wrapContentWidth().wrapContentHeight()) {
        Text(text = "Let's get started")
      }
    }
  }
}


@Preview(showBackground = true)
@Composable
fun fragment_intro_preview() {
    MyApplicationTheme {
        fragment_intro()
    }
}
